c3po/api/dto.py

```diff
@@ -1,11 +1,28 @@
 # Data Transfer Object- Responsible for carrying data between processes
 from flask import current_app
-from flask_restx import Namespace, fields
+from flask_restx import Namespace
+
+from marshmallow import Schema, fields
 
 
 class FeedDto:
     ns = Namespace('feed', description='Feed Related operations')
-    songObject = ns.model('songObject', {
-        'caption': fields.String,
-        'link': fields.String
-    })
+
+class PostDto(Schema):
+    caption = fields.String()
+    share_date = fields.DateTime()
+    likes_count = fields.Integer()
+
+class ArtistDto(Schema):
+    name = fields.String()
+    image = fields.String()
+
+class SongDto(Schema):
+    name = fields.String()
+    image = fields.String()
+
+
+post_dto = PostDto()
+artist_dto = ArtistDto()
+song_dto = SongDto()
+genre_dto = GenreDto()
```

Comment:
Where is `GenreDto()`?

Reply:
I'll remove it. I had created it earlier but then realized it was unnecessary :sweat_smile: